### ISSUE TITLE
Support object literal sugaring

### ISSUE THREAD

**Issue by matsko** [angular/ts2dart#297]
Typescript and ES6 allow for this:

```ts
var name = "matias";
var info = {matias};
console.log(info); // { name: "matias" }
```

Is there anyway we can support like this for the dart side of things via ts2dart?

**Comment by mprobst** [comment]
Yes. You'd need to add some code (here)[https://github.com/angular/ts2dart/blob/01420c047a3b0a7cdbb0dfcf213237f8204f8485/lib/literal.ts#L57] to handle the appropriate `SyntaxKind`, and explicitly emit the properties.

However (this code here)[https://github.com/angular/ts2dart/blob/01420c047a3b0a7cdbb0dfcf213237f8204f8485/lib/literal.ts#L77] looks as if we should already support them. What's the error you're getting?
